tests/_utils.py

```diff
@@ -63,3 +64,18 @@ def load_module(location):
 
 def is_callable(obj):
     return hasattr(obj, "__call__")
+
+
+def create_guides(image):
+    mask = torch.stack(
+        (
+            torch.ones(
+                [1, 1, ceil(image.size()[2] / 2), image.size()[3]], dtype=torch.bool
```

Comment:
If the image height is odd the guide size will mismatch since it is rounded up twice. I would do

```python
top_height = height // 2 # built in floor
bottom_height = height - top_height
```

This way you are safe